### ISSUE TITLE
Add in `ImportError` handeling

### ISSUE THREAD

**Issue by benhoff** [benhoff/simpleyapsy#40]
currently just `pass`ing.
https://github.com/benhoff/pluginmanager/blob/master/pluginmanager/module_manager.py#L109

**Comment by benhoff** [comment]
Here's what yapsy does...

```
 exc_info = sys.exc_info()
 log.error("Unable to import plugin: %s" % candidate_filepath, exc_info=exc_info)
 plugin_info.error = exc_info
 processed_plugins.append(plugin_info)
 continue
```

**Comment by benhoff** [comment]
Track progress in this pull request/branch: https://github.com/benhoff/pluginmanager/pull/66

**Comment by benhoff** [comment]
Fixed in fa79b1cf9aaf9722355e1c7ca368b864f5e1cd44
